addons/sale/controllers/combo_configurator.py

```diff
@@ -131,10 +131,14 @@ def _get_combo_item_data(
         """
         ptals_data = self._get_ptals_data(combo_item.product_id, selected_combo_item)
         # If the combo choice has only one combo item, and that combo item can't be configured (i.e.
-        # it has no `no_variant` attributes), then it should be preselected, as the user has to
-        # select it anyway.
-        is_preselected = len(combo.combo_item_ids) == 1 and not any(
-            ptal['create_variant'] == 'no_variant' for ptal in ptals_data
+        # it has no configurable `no_variant` attributes), then it should be preselected, as the
+        # user has to select it anyway.
+        is_preselected = (
+            len(combo.combo_item_ids) == 1
+            and not any(
+                ptal.attribute_id.create_variant == 'no_variant' and ptal._is_configurable()
```

Comment:
shouldn't it be an or here ? :thinking:

Reply:
A custom ptav would be configurable regardless of whether the attribute is no_variant or not no?

Reply:
Yeah, I realized that after posting my comment. I guess custom PTAVs in combos are not that common though?

FYI, I would need to fix this in other places too, so might keep it for a follow-up PR if that's ok with you.